### PR TITLE
Materialize: Add version 1.78

### PR DESCRIPTION
**Pull request by ddavness**
A program that converts images into materials that you can use in video game development.

### CODE REVIEW

File: bucket/materialize.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "homepage": "http://www.boundingboxsoftware.com/materialize/",
+    "license": "GPL-3.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "http://www.boundingboxsoftware.com/materialize/download/Materialize_1.78.zip",
+            "hash": "b0f6e3a905781e85dcd7db2383646bce51b8984ab93d1e3d13be42d5bea8904a"
+        }
+    },
+    "bin": "Materialize.exe",
+    "shortcuts": [["Materialize.exe", "Materialize"]],
```
Comment (Ash258) [inline bucket/materialize.json]: ...

```suggestion
    "shortcuts": [
        [
            "Materialize.exe",
            "Materialize"
        ]
    ],
```

File: bucket/materialize.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "suggest": {
+        "vcredist": "extras/vcredist2019"
```
Comment (Ash258) [inline bucket/materialize.json]: Link inside FAQ and download page lead into 2013 version.

File: bucket/materialize.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    "checkver": {
+        "url": "http://www.boundingboxsoftware.com/materialize/getkey.php",
+        "regex": "download/Materialize_(\\d+\\.\\d+).zip"
```
Comment (Ash258) [inline bucket/materialize.json]:
```suggestion
        "regex": "download/Materialize_([\\d.]+)\\.zip"
```

File: bucket/materialize.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.78",
+    "description": "Image-to-material converter for use in video games",
+    "homepage": "http://www.boundingboxsoftware.com/materialize/",
+    "license": "GPL-3.0-or-later",
```
Comment (Ash258) [inline bucket/materialize.json]: > GNU General Public License v3. Use it for whatever you want.

There is nothing about future versions

```suggestion
    "license": "GPL-3.0-only",
```

File: bucket/materialize.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.78",
+    "description": "Image-to-material converter for use in video games",
```
Comment (Ash258) [inline bucket/materialize.json]:
```suggestion
    "description": "Image-to-material converter for video game development",
```

File: bucket/materialize.json (hunk 4 above)
Comment (ddavness) [reply bucket/materialize.json]: I assumed that the "or later" statement was implicit. My bad

File: bucket/materialize.json (hunk 2 above)
Comment (ddavness) [reply bucket/materialize.json]: I'm going to test this before changing the VC version...

**Comment by ddavness** [comment]
Apparently the `Materialize_Data` folder isn't more than the program's libraries (Unity, Mono, etc.) and some obscure dev-side configs you probably shouldn't be touching anyway.

User settings are saved in a file called `settings.txt`, which is not there by default, and not in the `Materialize_Data` folder (that's the only thing we have got to persist...)

File: bucket/materialize.json (hunk 2 above)
Comment (ddavness) [reply bucket/materialize.json]: Update: It does indeed require 2013. 2015/17/19 won't do.

**Comment by ddavness** [comment]
There you go, persisting should work now.

File: bucket/materialize.json [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+        }
+    },
+    "bin": "Materialize.exe",
+    "shortcuts": [
+        [
+            "Materialize.exe",
+            "Materialize"
+        ]
+    ],
+    "pre_install": [
```
[the remark this answers is not in the document]
Comment (ddavness) [reply bucket/materialize.json]: Not a fan of one-liners, but ¯\_(ツ)_/¯

[the remark this answers is not in the document]
Comment (Ash258) [reply bucket/materialize.json]: This is not for oneliners. Main reason is that awefull write-output with redirection, when you have native command

**Pull request merged**